src/pages/create/forms/RulesForm/index.tsx

```diff
@@ -128,6 +149,30 @@ export default function RulesForm({
               <Trans>Allow token minting</Trans>
             </div>
           </Form.Item>
+
+          <Form.Item
```

Comment:
Adds this to the Rules form

<img width="623" alt="Screen Shot 2022-07-05 at 10 50 26 AM" src="https://user-images.githubusercontent.com/94939382/177231430-1e28d1c2-c06c-48f9-949a-cea2a495ef42.png">